pkg/workflows/internal/v2/sdkimpl/runtime.go

```diff
@@ -0,0 +1,110 @@
+package sdkimpl
+
+import (
+	"errors"
+	"io"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+	valuespb "github.com/smartcontractkit/chainlink-common/pkg/values/pb"
+	"github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk/v2"
+	"github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk/v2/pb"
+)
+
+type CallCapabilityFn func(request *pb.CapabilityRequest) ([]byte, error)
+type AwaitCapabilitiesFn func(request *pb.AwaitCapabilitiesRequest, maxResponseSize uint64) (*pb.AwaitCapabilitiesResponse, error)
```

Comment:
Does it make sense for the guest to always provide max response size? Why not enforce it in the host with a global default? We don't let guest provide other limits like timeout...